scss/foundation-styles/glyphs.scss

```diff
@@ -29,6 +29,33 @@
       height: 16px;
     }
   }
+
+  &.x-small {
+    &::before {
```

Comment:
https://github.com/mozilla/network-pulse/pull/1066#issuecomment-495024408
https://github.com/mozilla/network-pulse/pull/1066#issuecomment-495031415

If someone has larger fingers, it will be difficult for them to tap a 12x12 icon on mobile without tapping multiple icons at once. At desktop, the Github icon isn't legible at 12x12, but starts to become legible at around 13/14px. I made sure these changes were okay with @natalieworth.

Reply:
@natalieworth can you update the Sketch file to reflect ^? Thanks!

Reply:
pinging @natalieworth again 😉 I'm not following the convo regarding icon size but can you update Sketch file so they reflect to the icon sizes we decided on? Thanks!